r2r-rebaseline.cmd

```diff
@@ -0,0 +1,25 @@
+"C:\Repos\coreclr\Tools\dotnetcli\dotnet.exe" build /p:__BuildArch=x64 /p:__BuildOS=Windows_NT /p:__BuildType=Checked src\tools\r2rdump\R2RDump.csproj
```

Comment:
I'm not sure this belongs at the root of the repo. Can you place this in the r2rdump test directory?

Reply:
This was something I used for testing my changes and accidentally committed. I can clean it up and commit it, since it might be useful